test/indexing.jl

```diff
@@ -74,8 +74,8 @@
         m = [1.0 2.0; 3.0 4.0]
 
         @test m[(1,2), (1,2)] === @SMatrix [1.0 2.0; 3.0 4.0]
-        @test m[1, (1,2)] === @SVector [1.0, 2.0]
-        @test m[(1,2), 1] === @SVector [1.0, 3.0]
+        @test m[1, (1,2)] ===  (1.0, 2.0)
+        @test m[(1,2), 1] ===  (1.0, 3.0)
```

Comment:
Yeah I noted this as a todo ;) I don't really have a good answer, though :( Tuple of tuple would make sense, but who would like to work with that?!
I think in the end the only reasonable thing to do is to not allow it, and force the user to use some static array for indexing.